packages/dynamite/dynamite/lib/src/type_result/object.dart

```diff
@@ -8,14 +8,25 @@ class TypeResultObject extends TypeResult {
         );
 
   @override
-  String serialize(final String object) => '$object.toJson()';
+  String serialize(final String object) {
+    if (className == 'ContentString') {
```

Comment:
I think using a constant for this string would better

Reply:
what do you mean?

Reply:
Just make a constant variable somewhere containing `'ContentString'` so that you don't have to type it over and over. Might prevent some future typos.

Reply:
Yeah in theory the serialization under the contentstring can be used for all types. So no `TypeResult` would need to override it. I just didn't want to change to much already reviewed code :)

I'll change as suggested and we'll see after this finally lands :+1: